Dockerfile

```diff
@@ -15,4 +15,6 @@ EXPOSE 3000
 
 COPY . /code
 
+RUN [ "npm", "run", "build-css" ]
```

Comment:
As discussed in the c2s-dev catch up this will stay as is for now but the read me has been updated to reflect the need to run build-css when running the app locally.